empack_config.yaml

```
@@ -0,0 +1,42 @@
+packages:
+  python-dateutil:
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - pattern: '*dateutil-zoneinfo.tar.gz'
+    exclude_patterns:
+      - pattern: '**/tests/**/*.py'
+      - pattern: '**/tests/**/*.so'
+  matplotlib:
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - pattern: '*matplotlibrc'
+      - pattern: '**/matplotlib/mpl-data/**'
+      - pattern: '**/matplotlib/fontlist.json'
+    exclude_patterns:
+      - pattern: '**/tests/**/*.py'
+      - pattern: '**/tests/**/*.so'
+  scikit-learn:
+    include_patterns:
+      - pattern: '*.so'
+      - pattern: '*.py'
+      - pattern: '**/sklearn/datasets/**'
+    exclude_patterns:
+      - pattern: '**/tests/**/*.py'
+      - pattern: '**/tests/**/*.so'
+  scikit-image:
```

Shamelessly adding a new one :) It seems ipympl broke too recently

```suggestion
  ipympl:
    include_patterns:
      - pattern: '**/ipympl/static'
  scikit-image:
```

---

but we still need the *py files,right?

---

Yes, does it only include the matching patterns? It does not use the default ones?

---

the default is just for PKGs that are not explicitly listed. so you still need to add the stuff

---

first, the files are checked against the include-patterns, if any of these matches, the file is considered.
All considered files are checked against all exclude patterns, if none of the exclude pattern are matching, the file will be included

---

Ok :) I updated my suggestion accordingly